sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -2163,3 +2164,44 @@ def test_keys_with_specialchar(self, tables_storage_account_name, tables_primary
             self.table.delete_entity(entity1.copy())
         finally:
             self._tear_down()
+
+    @tables_decorator
+    @recorded_by_proxy
+    def test_entity_with_edmtypes(self, tables_storage_account_name, tables_primary_storage_account_key):
+        # Arrange
+        self._set_up(tables_storage_account_name, tables_primary_storage_account_key)
+        partition, row = self._create_pk_rk(None, None)
+
+        entity = {
+            'PartitionKey': partition,
+            'RowKey': row,
+            "bool": ("false", "Edm.Boolean"),
```

Comment:
Shouldn't the second value be EdmType.Boolean?

Reply:
I wanted to ensure that the enum was interchangable with the equivalent string value.
The CLI command will simply pass through the string, rather than convert it to an enum reference. And while that already worked, I don't think we had any tests for it.